checkQC/default_config/config.yaml

```diff
@@ -606,6 +606,103 @@ nextseq500_high:
       - name: Q30Handler
         warning: 80
         error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
+      - name: ErrorRateHandler
+        allow_missing_error_rate: False
+        warning: 1.5
+        error: unknown
+  151:
+      handlers:
+      - name: ClusterPFHandler
+        warning: 60
+        error: unknown
+      - name: Q30Handler
+        warning: 75
+        error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
+      - name: ErrorRateHandler
+        allow_missing_error_rate: False
+        warning: 1.5
+        error: unknown
+  301:
+      handlers:
+      - name: ClusterPFHandler
+        warning: 60
+        error: unknown
+      - name: Q30Handler
+        warning: 75
+        error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
+      - name: ErrorRateHandler
+        allow_missing_error_rate: False
+        warning: 1.5
+        error: unknown
+
+nextseq500_mid:
+  76:
+    handlers:
+      - name: ClusterPFHandler
+        warning: 60
+        error: unknown
+      - name: Q30Handler
+        warning: 80
+        error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
+      - name: ErrorRateHandler
+        allow_missing_error_rate: False
+        warning: 1.5
+        error: unknown
+  151:
+      handlers:
+      - name: ClusterPFHandler
+        warning: 60
+        error: unknown
+      - name: Q30Handler
+        warning: 75
+        error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
+      - name: ErrorRateHandler
+        allow_missing_error_rate: False
+        warning: 1.5
+        error: unknown
+  301:
+      handlers:
+      - name: ClusterPFHandler
+        warning: 60
+        error: unknown
+      - name: Q30Handler
+        warning: 75
+        error: unknown
+      - name: ReadsPerSampleHandler
+        warning: unknown
+        error: 15
+      - name: ErrorRateHandler
+        allow_missing_error_rate: False
+        warning: 1.5
+        error: unknown
+
+nextseq550_high:
+  76:
+    handlers:
+      - name: ClusterPFHandler
+        warning: 60
```

Comment:
I notice that you have set this the same for both mid and high, wouldn't it be expected that the expected number of clusters differ a bit? 

Referring to https://www.illumina.com/systems/sequencing-platforms/nextseq/specifications.html, I would set this threshold to 100 (high) and 32.5 (mid). Not sure about NextSeq500 since it it looks like Illumina doesn't have information about that sequencer anymore.

Reply:
I thought it was a percentage, this is an absolute number? If so I think the values you propose are correct and would match my experience.

Reply:
FTR, the sequencing hardware is the same as the 550, the only capability the 550 has over the 500 is the ability to process arrays.